docs/deployment/pull-backup.rst

```diff
@@ -48,19 +46,26 @@ the mount point prefix, e.g. /mnt/sshfs/bin/bash instead of /bin/bash).
     cd ~
     umount /mnt/sshfs
 
-Restoring a full backup
------------------------
+Restore methods
+---------------
+
+The counterpart of a pull backup is a push restore. When restoring from a
+backup, you might have to take a closer look on the user and group IDs and names
+of the backup content. Depending on the type of restore – full restore or
+partial restore – there are different methods to make sure the correct IDs are
+restored. Generally, there is no direct way for borg to know the correct UID:GID
+of files, because it uses the ``/etc/passwd`` and the ``/etc/group`` of the
+backup server.
 
-In case of a restore there is no direct way for borg to know the correct UID:GID
-of files, because it uses the ``/etc/passwd`` and ``/etc/group`` of the backup
-server. Using the archived UIDs/GIDs might lead to wrong results, because after
-a reinstall the name-to-ID mapping might have changed.
+Partial restore
+~~~~~~~~~~~~~~~
 
+In case of a partial restore, using the archived UIDs/GIDs might lead to wrong
+results, because after a reinstall the name-to-ID mapping might have changed.
```

Comment:
... wrong results if the name-to-ID mapping on the target system has changed compared to backup time (might be the case e.g. for a fresh OS install).